### PR TITLE
docs(troubleshooting): include documentation

### PR DESCRIPTION
**Pull request by kalinchernev**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,36 @@
+# Temporary files
+.tmp
+
 # Ignore some folders
+bower_components
```
Comment (yhuard) [inline .gitignore]: We don't use bower

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
 node_modules
 /static/framework
-/dist
+/static/build
+dist
+www
```
Comment (yhuard) [inline .gitignore]: `www` folder is not used anymore

File: .gitignore [hunk 3]
```diff
@@ -1,9 +1,36 @@
+# Temporary files
+.tmp
+
 # Ignore some folders
+bower_components
 node_modules
 /static/framework
-/dist
+/static/build
```
Comment (yhuard) [inline .gitignore]: `static/build` folder is not used anymore

File: .gitignore [hunk 4]
```diff
@@ -1,9 +1,36 @@
...
+.idea
+.c9
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
```
Comment (yhuard) [inline .gitignore]: We don't have any code coverage tool for now, so let's remove the following lines

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+## Issue [INNO-XXXX](https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-XXXX)
```
Comment (yhuard) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not sure about this one. When you have breaking changes in your PR, your PR message should start with `BREAKING CHANGE:`

**Comment by kalinchernev** [comment]
thanks, @yhuard updates applied

File: .gitignore [hunk 6]
```diff
@@ -23,12 +20,6 @@ pids
 *.seed
 *.pid.lock
 
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
 # nyc test coverage
```
Comment (yhuard) [inline .gitignore]: Still useless 😉

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,3 +1,3 @@
-## Issue [INNO-XXXX](https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-XXXX)
+## Issue
```
Comment (yhuard) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'm not confident with PULL_REQUEST_TEMPLATE.md, maybe @degliwe could share his thoughts

**Comment by yhuard** [comment]
Thanks for your effort!

**Pull request merged**